From 5b21444adc56e95ed98301440a9265912a8f3ee2 Mon Sep 17 00:00:00 2001
From: Etaash Mathamsetty
 <45927311+Etaash-mathamsetty@users.noreply.github.com>
Date: Sat, 13 Dec 2025 21:45:19 -0600
Subject: [PATCH 414/423] HACK: wine.inf: Allow builtin for vulkan-1 in RDR2.

D3D12 mode now works, and native preference should make the vulkan mode continue to work if the dll is present.
---
 loader/wine.inf.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/loader/wine.inf.in b/loader/wine.inf.in
index 3aeb6f6e31f..8ee33414252 100644
--- a/loader/wine.inf.in
+++ b/loader/wine.inf.in
@@ -3026,7 +3026,7 @@ HKCU,Software\Wine\AppDefaults\ffxvi_demo.exe\DllOverrides,"atiadlxx",,"builtin"
 HKCU,Software\Wine\AppDefaults\FarCry5.exe\DllOverrides,"atiadlxx",,"builtin"
 HKCU,Software\Wine\AppDefaults\rayne1.exe\DllOverrides,"d3d8",,"native"
 HKCU,Software\Wine\AppDefaults\rayne2.exe\DllOverrides,"d3d8",,"native"
-HKCU,Software\Wine\AppDefaults\RDR2.exe\DllOverrides,"vulkan-1",,"native"
+HKCU,Software\Wine\AppDefaults\RDR2.exe\DllOverrides,"vulkan-1",,"native,builtin"
 HKCU,Software\Wine\AppDefaults\ActOfWar.exe,"Version",,"winxp"
 HKLM,Software\Khronos\OpenXR\1,"ActiveRuntime",,"C:\openxr\wineopenxr64.json"
 HKLM,Software\Wow6432Node\lucasarts entertainment company llc\Star Wars: Episode I Racer\v1.0,"Display Height",0x10001,480
-- 
2.52.0

